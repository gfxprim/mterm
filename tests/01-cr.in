10 1
abc\r
