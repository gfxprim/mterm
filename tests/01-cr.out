 ----------
|abc       |
 ----------
size 1x10 cursor 0x0
